htlcswitch/switch.go

```diff
@@ -85,6 +86,9 @@ var (
 	// fail payments if they increase our fee exposure. This is currently
 	// set to 500m msats.
 	DefaultMaxFeeExposure = lnwire.MilliSatoshi(500_000_000)
+
+	// background is a shortcut for context.Background.
+	background = context.Background()
```

Comment:
i dont think we should do this. Rather use a `context.TODO()` where needed

Reply:
if you rebase on top of https://github.com/lightningnetwork/lnd/pull/9344, then we can also add a context guard here and then we only need a single context.TODO() in `Start()`